simulation/agents.py

```diff
@@ -686,3 +686,6 @@ def make_decisions(self, model):
             self.heating_system_costs_subsidies = costs_subsidies
             self.heating_system_costs_insulation = costs_insulation
             self.insulation_element_upgrade_costs = chosen_insulation_costs
+
+        if self.is_heat_pump_aware:
+            model.households_heat_pump_aware_at_current_step += 1
```

Comment:
Is this attribute correct? I'm not seeing this method being declared anywhere...

e.g. `heat_pump_installations_at_current_step` is declared in collectors.py and accessed in model.py

I did see you've created the method  `model_heat_pump_awareness` though.. are you missing some suffix?